packages/validators-util/src/isValidRegistreringsnummer/isValidRegistreringsnummer.test.ts

```diff
@@ -2,10 +2,34 @@ import { isValidRegistreringsnummer } from "./isValidRegistreringsnummer";
 
 describe("isValidRegistreringsnummer", () => {
     it("should return true for two letters and 5-digit numbers", () => {
-        expect(isValidRegistreringsnummer("AB12345")).toBe(true);
+        expect(isValidRegistreringsnummer({ value: "AB12345" })).toBe(true);
     });
 
     it("should return false if only one letter and more than 5-digit numbers", () => {
-        expect(isValidRegistreringsnummer("A1234567")).toBe(false);
+        expect(isValidRegistreringsnummer({ value: "A1234567" })).toBe(false);
+    });
+
+    it("should return false for two letters and 4-digit numbers", () => {
+        expect(isValidRegistreringsnummer({ value: "AB1234" })).toBe(false);
```

Comment:
Veteranbiler kan vel i teorien ha en kombo som dette, men de går kanskje som en egen kategori?

Reply:
Ikke mulig å kjøpe det på nett, derfor har vi ikke noe validering for det. Tenker at dersom noen trenger det så får de heller lager en PR på det slik at det blir rett.